proto/feature/feature.proto

```diff
@@ -53,6 +53,7 @@ message Feature {
   bool archived = 20;
   repeated Prerequisite prerequisites = 21;
   string sampling_seed = 22;
+  repeated ScheduleUpdate schedule_update = 23;
```

Comment:
Because the user might want to schedule multiple times, I'm using an array, so when implementing it on the frontend, we won't need to make changes on the backend.